src/Four_Player_Adapter.md

```diff
@@ -5,41 +5,136 @@ to connect for multiplayer via [serial data transfer](./Serial_Data_Transfer_(Li
 The device is primarily designed for DMG consoles, with later models
 requiring Link Cable adapters.
 
-## Communication
+## Power
+
+The DMG-07 will not power on until it's Player 1 cable is plugged into
+a Game Boy link port to supply power. The Player 1 cable is the only one
+permanently attached to the device and has the power pin connected
+unlike typical link port cables.
+
+## Communication Phases
 
 The DMG-07 protocol can be divided into 2 sections, the "ping" phase, and
 the "transmission" phase. The initial ping phase involves sending packets
 back and forth between connected Game Boys probing for their current
-connection status. Afterwards, the DMG-07 enters into transmission mode
-where the Game Boys exchange data across the network.
+connection status. Afterwards, the DMG-07 is switched to the transmission
+phase where the Game Boys exchange data across the link cable network.
+Later, connected Game Boys may restart the ping phase by sending specific
+commands.
+
+An important thing to note is that all Game Boys transfer data across
+the DMG-07 in external clock mode (bit 0 of [SC] set to 0) with
```

Comment:
Following the convention from https://gbdev.io/pandocs/Serial_Data_Transfer_(Link_Cable).html
```suggestion
the DMG-07 in external clock mode (bit 0 of the SC register set to 0) with
```